www/_includes/layouts/menu.njk

```diff
@@ -60,19 +60,16 @@
     </li>
 
     <li>
-      <span class="menu-section-header">Usage</span>
+      <span class="menu-section-header">Reference</span>
       <ol>
         <li>
-          <a href="/configuration">CLI</a>
+          <a href="/reference/features">Features</a>
```

Comment:
Mind keeping the existing items in this section? A part of that was aspirational in the organization that I'd like us to match our current content to. We'll need to write a page for the JS API, and seperate the current "Creating Your Own Plugin" page into 1 guide & 1 reference (currently it contains both).